app/static/settings/languages.json

```
@@ -46,4 +47,6 @@
   {"name": "Turkish (Türk)", "value": "lang_tr"},
   {"name": "Ukranian (Український)", "value": "lang_uk"},
   {"name": "Vietnamese (Tiếng Việt)", "value": "lang_vi"}
```

Need to add a comma after this line.

```suggestion
  {"name": "Vietnamese (Tiếng Việt)", "value": "lang_vi"},
```